lib/server.js

```diff
@@ -66,6 +67,9 @@ function CoAPServer(options, listener) {
     this._options.piggybackReplyMs = parameters.piggybackReplyMs
   }
 
+  if (!isBoolean(this._options.sendAcksForNonConfirmablePackets)) {
+    this._options.sendAcksForNonConfirmablePackets = parameters.sendAcksForNonConfirmablePackets
+  }
```

Comment:
just to be sure:
```
  if (!isBoolean(this._options.sendAcksForNonConfirmablePackets)) {
    this._options.sendAcksForNonConfirmablePackets = parameters.sendAcksForNonConfirmablePackets
  }
```

if the server options is invalid, see test case "define invalid sendAcksForNonConfirmablePackets setting".